File: app/src/main/java/com/cs389team4/needtofeed/ui/ActiveOrderActivity.java
package com.cs389team4.needtofeed.ui;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.cs389team4.needtofeed.databinding.ActivityActiveOrderBinding;
import com.mapbox.maps.MapView;
import com.mapbox.maps.Style;

public class ActiveOrderActivity extends AppCompatActivity {
    MapView mapView = null;

    ActivityActiveOrderBinding binding;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivityActiveOrderBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        mapView = binding.activeOrderMap;
        mapView.getMapboxMap().loadStyleUri(Style.MAPBOX_STREETS);
    }
}

File: app/src/main/java/com/cs389team4/needtofeed/ui/ActiveOrderActivity.kt
package com.cs389team4.needtofeed.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.cs389team4.needtofeed.databinding.ActivityActiveOrderBinding

class ActiveOrderActivity : AppCompatActivity() {
    private lateinit var binding: ActivityActiveOrderBinding

    private val bottomSheetFragment = ActiveOrderBottomSheetFragment();

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityActiveOrderBinding.inflate(layoutInflater)

        setContentView(binding.root)

        bottomSheetFragment.isCancelable = true
        bottomSheetFragment.show(supportFragmentManager, bottomSheetFragment.tag)
    }
}
